### PR TITLE
Connect schedule to API.

### PR DESCRIPTION
**Pull request by yonimn2000**
It works for the current week but I could not figure out how to make the schedule week itself change after selecting a different week from the list. The thing I tried did not work. Not too familiar with React states yet, I guess...

Also, the table headers are not showing up for some reason. So yeah, help wanted.

image

### CODE REVIEW

**Comment by sterlingwalker** [comment]
The reason the table wasn't updating is that you were referencing an eventhandler that wasn't connected with `employeeSchedule`. Also the `getSchedule` had to be able to be called repeatedly. `useEffect()` is nice because it runs under certain scenarios such as after the component mounts, but you cannot call it directly.

File: src/api.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { employees } from './demo';
-const apiPath = '/hr-server/api/v1/';
+const serverHostname = 'http://localhost:8095';
```
Comment (sterlingwalker) [inline src/api.js:2]: So in the `package.json` I set up a proxy that has this string already.

**Comment by yonimn2000** [comment]
Perfect. Thank you @sterlingwalker ! I appreciate you adjusting the code to fix the issues.

**Review by sterlingwalker — APPROVED** [review]
Very good! Thanks so much 😎!

**Pull request merged**